source/elements/oneDPL/source/parallel_api/parallel_range_api.rst

```diff
@@ -33,6 +33,9 @@ The following differences to the standard C++ range algorithms apply:
   In that case, the returned value contains iterators pointing to the positions past the last elements
   processed according to the algorithm semantics.
 - ``for_each`` does not return its function object.
+- ``reverse_copy`` returns ``std::ranges::in_in_out_result`` rather than its alias,
+  ``std::ranges::reverse_copy_truncated_result``, which is proposed for C++26 in
+  `P3709R2 <https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2025/p3709r2.html>`_.
```

Comment:
I think calling it "its alias" may be slightly misleading because `reverse_copy_truncated_result` is not _exactly_ the same as `in_in_out_result`, in that it takes only 2 iterator types and replicates the first twice when using `in_in_out_result`.

```suggestion
- ``reverse_copy`` returns ``std::ranges::in_in_out_result`` rather than the alias,
  ``std::ranges::reverse_copy_truncated_result``, which is proposed for C++26 in
  `P3709R2 <https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2025/p3709r2.html>`_.
```